docs/sources/installation/windows.md

```diff
@@ -43,6 +43,26 @@ is developed, you can launch only Linux containers from your Windows machine.
 
 ![](/installation/images/windows-boot2docker-start.png)
 
+##  Installing Docker client (without Boot2Docker)
```

Comment:
@ahmetalpbalkan We are moving to a policy where we don't document other project's installers or package manager installs.  There are several reasons for this:

* It puts a burden on us to test 
* It puts a burden on us to update

Happy to put a mention in a references section.

## References

If you have Docker hosts running and if you don't wish to do a Boot2Docker installation, you can install the `docker.exe` using unofficial Windows package manager Chocolately. For information on how to do this, see (instructions at [chocolatey.org](http://chocolatey.org)).